src/api/status/env.local

```diff
@@ -1 +0,0 @@
-PLANET_PORT=1111
```

Comment:
why is this line removed?

Reply:
Hi, I got a comment here from @humphd. I think this is not necessary.
https://github.com/Seneca-CDOT/telescope/pull/2476#discussion_r752791610

Reply:
That's actually a new line unnecessary not the `PLANET_PORT`

Reply:
I just realized that, Added back the <code>env.local</code>